datafusion/core/src/datasource/physical_plan/parquet/row_groups.rs

```diff
@@ -854,9 +956,9 @@ mod tests {
         let rgm2 = get_row_group_meta_data(
             &schema_descr,
             vec![ParquetStatistics::fixed_len_byte_array(
-                // 5.00
+                // 10.00
```

Comment:
can you explain why you changed this value in the test? 

Would it be possible to change this PR  to not change existing tests so it is clear that the code change in this PR doesn't cause a regression in existing behavior?  Maybe we can add a new test case with the different values?

Reply:
It's main purpose is to prune the **_rgm2_** by filter `c1 in (10, 300, 400)`. If it's a concern, maybe I can introduce another independent new test case for it.